src/IssueViz.Security/OpenInIDE/Api/IHotspotToIssueVisualizationConverter.cs

```diff
@@ -0,0 +1,94 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.ComponentModel.Composition;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+using SonarLint.VisualStudio.IssueVisualization.Security.Models;
+using SonarQube.Client.Models;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Api
+{
+    internal interface IHotspotToIssueVisualizationConverter
+    {
+        IAnalysisIssueVisualization Convert(SonarQubeHotspot sonarQubeHotspot);
+    }
+
+    [Export(typeof(IHotspotToIssueVisualizationConverter))]
+    internal class HotspotToIssueVisualizationConverter : IHotspotToIssueVisualizationConverter
+    {
+        private readonly IAnalysisIssueVisualizationConverter issueVisualizationConverter;
+
+        [ImportingConstructor]
+        public HotspotToIssueVisualizationConverter(IAnalysisIssueVisualizationConverter issueVisualizationConverter)
+        {
+            this.issueVisualizationConverter = issueVisualizationConverter;
+        }
+
+        public IAnalysisIssueVisualization Convert(SonarQubeHotspot sonarQubeHotspot)
+        {
+            var hotspot = ConvertToHotspot(sonarQubeHotspot);
+            var issueViz = issueVisualizationConverter.Convert(hotspot);
+
+            return issueViz;
+        }
+
+        private Hotspot ConvertToHotspot(SonarQubeHotspot sonarQubeHotspot)
+        {
+            // todo: calculate file path
+            var filePath = sonarQubeHotspot.ComponentPath;
```

Comment:
Since the file path is not translated, this will be the value that is shown in the data table.
When we attempt to navigate to it, LocationNavigator should fail because the file doesn't exist. The span should be then set to Empty and we could mark the location as non-navigable.